### DISCUSSION TITLE
Permission System feels kinda wierd

### DISCUSSION THREAD

**Discussion by T2brozz** [outline/outline#7808] [General] [answered]
Hi there,

We switched from Wiki JS to Outline., and so far, everything works great. 
But the Permission feels a little weird to use. 

So I have a collection where Group A has edit Permissions on a collection. So there can't manage the Permissions of a Collection. 
But a Document or a Folder there can share with a User not in the Group? Also, they can allow that User to Manage the Resource?  

Yes, there are probably situations where this is desired, but why can an Editor give a User higher permissions?

Sharing a Document is okay I guess, but I Like that a Person who I share a document with can't share it further.

**Comment by tommoor** [comment, marked as answer]
This sounds like the discussion happening here: https://github.com/outline/outline/issues/7229
